### DISCUSSION TITLE
useOutsideClick usage

### DISCUSSION THREAD

**Discussion by elreeda** [chakra-ui/chakra-ui#8301] [Q&A]
I want to show a suggestion for an input but it don't to be working as expected. [example ](https://play.chakra-ui.com/playground?code=N4Igxg9gJgpiBcICWBbADhATgFwAQCUYBDMPAM0whVwB0RNjS6BuGgO1Qx12AFcBnGAHle2fklgBhADZIwAawC+uClVogAAmAAWReZiIBaXkgD0DEthbsbbGAA8u5Xm1JIIbXAEE0aABQAlDzsuLiQbPx4ANpI-ABiEGACMFAANLiC2ACS8YnJUAC6uAC8BIzYAHTJAMrYRNgwfmRE0oIBrJ5hHpG4DGQlZZZVgoRkfmy80tLttqHJImISMDJy8n7AIaG9MGTw22Spm6G6bFDSMJh7gSUAfBkw2blJglBNLW2HnYoznQzYvJhPH4jrgADxQJAANzC0iI-H4ADkiCgYMU6D40HQbiDQqDtABGG4ACRgUwguEk0Bg1SIpwARhB7KDTATsZ0tmCkGw0KJcNgAJ5oVF0Br2KwgDIC87FYDABmYWCXXAAcnxaHsGQgsigysUyg8CWeMuuxTumRyhvyfmwmF4MACylMbI5oWAsUtL1wAH5cMD2S7QZN9jK+opnS6OaDZDd8czoziA9GAExxpDhiO46MAZlT6YDpkmedCQT2EymihBzIhkPDPwrbBAiiAA)is pretty straight forward.
am I missing something?

**Comment by segunadebayo** [comment]
`useOutsideClick` was removed. Use `useClickAway` from `react-use`.
